components/collapsible-panel/collapsible-panel.js

```diff
@@ -309,12 +315,14 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 	}
 
 	render() {
+		console.log();
 		const classes = {
 			'd2l-collapsible-panel': true,
 			'focused': this._focused,
 			'has-summary': this._hasSummary,
 			'has-before': this._hasBefore,
 			'scrolled': this._scrolled,
+			'group-last': this.parentElement.nodeName === 'D2L-COLLAPSIBLE-PANEL-GROUP' && this === this.parentElement.lastElementChild,
```

Comment:
One of the things that this doesn't work well for is - this panel won't be aware when panels are added or removed from the group. For that, it is better that the group listens for `slotchange` and then applies a class to the children. I think that's what our menu component does.

Reply:
Sounds good, I'll look at the implemenation for `menu`